src/util/getSortPreference.js

```diff
@@ -0,0 +1,8 @@
+// util
+import {
+  getSetting
+} from '../util.js'
+
+export const getSortPreference = contextMeta => {
```

Comment:
Functions should always have a JSDOC comment with a short description of the purpose of the function. In this case, it should explain the logic of the sort preference.